src/courses/templates/courses/chart_mark_distribution.html

```diff
@@ -1,105 +1,106 @@
-<script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/Chart.js/2.5.0/Chart.min.js"></script>
```

Comment:
This is still all js in an html file. Didn't resolve this?

Reply:
thought you wanted chart,js in static.
Still cant do it since template tags doesnt seem to work with static files

Reply:
changed chart.js back to cdn

Reply:
Sorry, miscommunication.